src/components/MessageBox.js

```diff
@@ -11,56 +12,55 @@ const StyleUtils = require('../utils/Style');
 
 class MessageBox extends React.Component {
   static propTypes = {
-    children: PropTypes.node,
+    button: PropTypes.node,
     color: PropTypes.string,
     icon: PropTypes.string,
+    isSmall: PropTypes.bool,
+    message: PropTypes.string,
     styles: PropTypes.object,
     theme: themeShape,
     title: PropTypes.string
   };
 
-  state = {
-    isOpen: true
-  };
-
-  _toggleMessageBox = () => {
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
-  };
-
   render () {
     const theme = StyleUtils.mergeTheme(this.props.theme);
     const styles = this.styles(theme);
 
     return (
-      <div className='mx-message-box' style={styles.component}>
-        <div onClick={this.props.children ? this._toggleMessageBox : () => {}} style={styles.header}>
-          <div style={styles.leftHeader}>
-            <Icon
-              size={20}
-              style={Object.assign({}, styles.icon, { marginRight: theme.Spacing.SMALL })}
-              type={this.props.icon}
-            />
-            <div style={styles.title}>{this.props.title}</div>
+      <article
+        aria-live='polite'
+        className='mx-message-box'
+        role='region'
+        style={styles.component}
+      >
+        <div style={styles.alertbar} />
+        <div style={styles.messageWrapper}>
+          <div style={styles.messageBody}>
+            <section>
+              <Icon
+                size={20}
+                style={styles.icon}
+                type={this.props.icon}
+              />
+            </section>
+            <section style={styles.messageContent}>
+              <p style={styles.title}>
+                {this.props.title}
+              </p>
+              { this.props.message ?
+                (<p role='status' style={styles.message}>
```

Comment:
😢 sorry to comment again....

I think we better move the `role='status'` here back up to the `messageContent` section and verify that voice over still works.

Reply:
I should have caught this myself, so no worries. I updated this, and tested.